src/EditorFeatures/Test2/IntelliSense/StrongNameKeyFileFixture.vb

```diff
@@ -0,0 +1,43 @@
+﻿Imports System.IO
+
+Namespace Microsoft.CodeAnalysis.Editor.UnitTests.IntelliSense
+    Public Class StrongNameKeyFileFixture
```

Comment:
This is a helper class that resembles the same behavior as [this class](https://github.com/dotnet/roslyn/blob/master/src/Test/Utilities/Portable/TempFiles/DisposableFile.cs) used in [this test](https://github.com/dotnet/roslyn/blob/master/src/Compilers/CSharp/Test/Emit/Attributes/InternalsVisibleToAndStrongNameTests.cs#L309). See my comment from [7 days ago](https://github.com/dotnet/roslyn/pull/19805#issuecomment-306330358) why creating a temp file was necessary. The class is named *StrongNameKeyFileFixture* because I wanted to use the [xunit class fixture feature](https://xunit.github.io/docs/shared-context.html#class-fixture) but couldn't use it because xunit isn't referenced by the test-project.